### ISSUE TITLE
Make the robot eject tips in different area of the bin

### ISSUE THREAD

**Issue by RussHobbs** [theosanderson/PlasmoTron#13]
Because ejecting used-tips happens from the same position, the tips piles up and it happens that one get stuck on top of another blocking the ejector. Could be solved by making the robot eject every N tips in a quarter of the bin.

**Comment by fschwach** [comment]
Good point. I'll deal with this. the only problem is that the trash is going to be different for different users, i.e. there is no standard size for this. But we can say that it has to be at least a minimum width and height and then apply a bit of randomness to the actual point at which it ejects, which should prevent the pileup.

**Comment by theosanderson** [comment]
Thanks Frank, do you plan to confine the footprint of this to
queueprocessor (and maybe equipment)? That would be my instinct
